Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/IndexMetricsParserBaselineTest.cs

```
@@ -0,0 +1,214 @@
+                  query: "SELECT VALUE STARTSWITH(r['Быстрая коричневая лиса прыгает через ленивую собаку'], 's', true) FROM root r"
+                ),
+
+                new IndexMetricsParserTestInput
+                (
+                  description: "Unicode Japanese",
+                  query: "SELECT VALUE STARTSWITH(r['素早く茶色のキツネが怠惰な犬を飛び越えます'], 's', true) FROM root r"
+                ),
+
+                new IndexMetricsParserTestInput
+                (
+                  description: "Unicode Hindi",
+                  query: "SELECT VALUE STARTSWITH(r['तेज, भूरी लोमडी आलसी कुत्ते के उपर कूद गई'], 's', true) FROM root r"
+                ),
+            };
+
+            this.ExecuteTestSuite(inputs);
+        }
+
+        public override IndexMetricsParserTestOutput ExecuteTest(IndexMetricsParserTestInput input)
+        {
+            try
+            {
+                QueryRequestOptions requestOptions = new QueryRequestOptions() { PopulateIndexMetrics = true };
+
+                FeedIterator<CosmosElement> itemQuery = testContainer.GetItemQueryIterator<CosmosElement>(
+                    input.Query,
+                    requestOptions: requestOptions);
+
+                // Index Metrics is returned fully on the first page so no need to worry about result set
+                FeedResponse<CosmosElement> page = itemQuery.ReadNextAsync().Result;
+                Assert.IsTrue(page.Headers.AllKeys().Length > 1);
+                Assert.IsNotNull(page.Headers.Get(HttpConstants.HttpHeaders.IndexUtilization), "Expected index utilization headers for query");
+                Assert.IsNotNull(page.IndexMetrics, "Expected index metrics response for query");
+
+                return new IndexMetricsParserTestOutput(page.IndexMetrics);
+            }
+            catch (Exception e)
+            {
+                return new IndexMetricsParserTestOutput(String.Empty, LinqTestsCommon.BuildExceptionMessageForTest(e));
```

Is this hit? Did I miss it in the baseline?

---

If not, please remove try...catch and let the exception bubble up (and fail the test).

---

It's not hit in these test cases because the exception is in case there is wrong with any of the execution, (but there shouldn't). I'm not sure there is even a way to force it to fail.

This effectively also fail the test too because if the catch is ever hit, it will also change the baseline, which in turns fail the comparison for the baseline, which fails the test. So I think we should keep it as is as a try catch

---

after discussion, we have decided to remove the try-catch